operators/src/pro/cache/tile_cache.rs

```diff
@@ -52,20 +462,18 @@ impl TileCache {
             });
         }
 
+        let cache_size_bytes =
```

Comment:
regarding the check above this line: maybe we should also check that the ratio is < 1

Reply:
0789dd4330c15bc9f11f40c260d8493037b4c1ee